qiskit_braket_provider/providers/braket_job.py

```diff
@@ -135,15 +136,31 @@ def cancel(self):
             task.cancel()
 
     def status(self):
-        braket_tasks_states = [task.state() for task in self._tasks]
+        braket_jobs_states = [task.state() for task in self._tasks]
```

Comment:
I think `braket_tasks_states` is valid. Is there a reason we switched this to `braket_jobs_states`?

Reply:
I must have changed it by mistake, this has now been resolved in the latest commit https://github.com/qiskit-community/qiskit-braket-provider/pull/105/commits/ec061fdba801c292de8319b1465a1dec45117371